docs/sources/reference/stdlib/encoding.md

```diff
@@ -27,6 +27,29 @@ into the original string.
 tangerine
 ```
 
+## encoding.to_base64
+
+The `encoding.to_base64` function encodes a string to RFC4648-compliant base64 string
+into the original string.
+
+### Examples
+
+```
+> encoding.to_base64("foobar123!?$*&()'-=@~")
```

Comment:
We should avoid using foo, bar, baz, foobar, etc. [developers.google.com/style/word-list#foo](https://developers.google.com/style/word-list#foo) You can use examplestring or something more generic.